app/components/search-facet-registration-type.js

```diff
@@ -43,16 +48,22 @@ export default Component.extend({
     }),
     init() {
         this._super(...arguments);
-        this.registrationTypes = [
-            'AsPredicted Preregistration',
-            'Election Research Preacceptance Competition',
-            'Open-Ended Registration',
-            'OSF-Standard Pre-Data Collection Registration',
-            'Prereg Challenge',
-            'Pre-Registration in Social Psychology (van \'t Veer & Giner-Sorolla, 2016): Pre-Registration',
-            'Replication Recipe (Brandt et al., 2013): Post-Completion',
-            'Replication Recipe (Brandt et al., 2013): Pre-Registration',
-        ];
+        this.get('store')
+            .findAll('metaschema')
+            .then(this._returnResults.bind(this))
+            .catch(this._errorMessage.bind(this));
+    },
+    _returnResults(results) {
+        const typeArr = [];
+        results.forEach(function(result) {
+            typeArr.push(result.get('name'));
+        });
+        typeArr.push('Election Research Preacceptance Competition');
```

Comment:
This makes me sad, but my guess is that when they deprecated it was removed from the endpoint, eh?

Can you put in a comment that says why this is being force added? (deprecated metaschema was removed from API but still needs to show up in search facets blah blah blah)